cypress/integration/userCanLoginForPremiumContent.feature.js

```diff
@@ -0,0 +1,136 @@
+describe("successfully", () => {
+  beforeEach(() => {
+    cy.server();
+    cy.route({
+      method: "GET",
+      url: "http://localhost:3000/api/articles",
+      response: "fixture:article_list.json",
+    });
+    cy.route({
+      method: "POST",
+      url: "http://localhost:3000/api/auth/*",
+      response: "fixture:successful_login.json",
+      headers: {
+        uid: "user@mail.com",
+      },
+    });
+    cy.route({
+      method: "GET",
+      url: "http://localhost:3000/api/auth/*",
+      response: "fixture:successful_login.json",
+      headers: {
+        uid: "user@mail.com",
+      },
+    });
+    cy.route({
+      method: "GET",
+      url: "http://localhost:3000/api/articles/3",
+      response: "fixture:premium_article.json",
+      headers: {
+        uid: "user@mail.com",
+      },
+    });
+    cy.visit("/");
+    cy.get("button#login").click();
+    cy.get("#login-form").within(() => {
+      cy.get("#email").type("user@mail.com");
+      cy.get("#password").type("password");
+      cy.get("Button#submit").contains("Submit").click();
+    });
+  });
+
+  it("and is directed to main page", () => {
+    cy.get("#article-1").should("contain", "title 1");
+  });
+
+  it("with valid credentials", () => {
+    cy.get("div#login").within(() => {
+      cy.get("p").should("contain", "user@mail.com");
+    });
+  });
+
+  it("can view premium article", () => {
+    cy.visit("/article/3");
+    cy.get("#article-3-body").then((text) => {
+      text.length >= 100;
```

Comment:
The amount of characters in the snippet is updated to 300.

Reply:
done, updated the fixture as well as it was 284 in total